pykokkos/core/cpp_setup.py

```diff
@@ -209,6 +209,9 @@ def invoke_script(self, output_dir: Path, space: ExecutionSpace, enable_uvm: boo
         if space is ExecutionSpace.Cuda:
             if enable_uvm:
                 view_space = "Kokkos::CudaUVMSpace"
+        if space is ExecutionSpace.HIP:
+            if enable_uvm:
+                view_space = "Kokkos::Experimental::HIPHostPinnedSpace"
```

Comment:
hmm ... this will mean, that it will not be comparable to cuda ... should we hold it until we have `3.7`?